adventurous-anteaters/adv_ant/retroNet/templates/account_deletion_confirmation.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'base.html' %}
+{% block content %}
+<div class="window-body"
+    style="background-color: #dfdfdf;margin-left: 10px;margin-right: 10px;font-family:Pixelated MS Sans Serif;">
```

Comment:
This is quite a lot of inline style. Why? Couldn't you have just added this to the `window-body` class? Seems messy.